script/Ciel/Day 14/14_01B_01_CIEL14B_1F.txt

```diff
@@ -292,7 +292,7 @@ The blood-rending――almost bone-breaking moment in time stretches on.
 -- 掃射されるアサルトライフル、同時展開される魔術式、相打ち覚悟の蛇腹剣。
 // 蛇腹剣 initially introduced as 'serpentine whip sword' in 12516.
 The strafing of Senpai's assault rifle.
- The simultaneous deployment of her Magecraft Formulae.
+ The simultaneous deployment of her Magecraft Formulas.
```

Comment:
Kinda prefer the latin plural for the more archaic flair

Reply:
I left it formulae for when Roa is talking but maybe just formulas for Shiki?

Reply:
Roa's in his head so let's go with latin